content/en/docs/tasks/administer-cluster/migrating-to-cloud-controller-manager.md

```diff
@@ -0,0 +1,175 @@
+---
+reviewers:
+- andrewsykim
+title: Migrating to Cloud Controller Manager
+content_template: templates/task
+min-kubernetes-server-version: v1.16
+---
+
+{{% capture overview %}}
+
+Cloud providers used to be integrated into the Kubernetes Controller Manager (KCM).
+In Kubernetes 1.17, this changes, and cloud platform integrations must run in a
```

Comment:
Instead of `Kubernetes 1.17` we should say `In a future Kubernetes release`.

Reply:
We prefer to [avoid statements about the future])(https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) - if it's feasible to document the “current state”, if you like, and not mention the project ambitions to do even better, that's typically better.

Sometimes you need to mention these details for the page to make sense and in those cases it's fine - it's a style _guide_ not a style rulebook.